xml/ay_docupdates.xml

```diff
@@ -30,6 +30,24 @@ https://etherpad.nue.suse.com/p/doc_team_meeting_agenda_2018-07-24-->
   <title>&productname; 15 SP0</title>
   <para/>
 
+  <sect2 xml:id="sec.ay.docupdates.sle15.sp0.jan_2019">
```

Comment:
You can remove this section from the ay_docupdates.xml file because we decided to skip the docupdate appendices for SLE 15 and the upcoming SP. (Sorry for that, you could not know, especially since our default PR form still asks for docupdate entries - I will try to adjust the form). However, for the SLE 12 product family (all maintenance branches for SLE12 and higher) , docupdate entries are still required.

Reply:
I kep the info in (in case we should enable the docupdate index again at a later point in time), but tweaked it a bit (used xref to section ID as term, added some markup, removed one detail).